src/Api/Workspaces.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of Bitbucket API Client.
+ *
+ * (c) Graham Campbell <graham@alt-three.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Bitbucket\Api;
+
+class Workspaces extends AbstractApi
+{
+    /**
+     * @param array $params
+     *
+     * @throws \Http\Client\Exception
+     *
+     * @return array
+     */
+    public function list(array $params = [])
+    {
+        $path = $this->buildWorkspacePath();
+
+        return $this->get($path, $params);
+    }
+
+    public function all()
```

Comment:
As before, we don't need these all methods. We already have this implemented in a way that respects separation of concerns.